### DISCUSSION TITLE
Live Stream Help

### DISCUSSION THREAD

**Discussion by BGriffy78** [gopro/OpenGoPro#296] [Q&A]
Hi All,

Warning, I'm a newcomer... Been having fun for a couple of weeks dabbling with my GoPro and python scripts, but these are not advanced questions!!

I am struggling to get live stream working with my Hero 10.  I have tried using the livestream.py demo code and the gui_demo.

I suspect my issue is not understanding how the URL is to be formatted, and probably not understanding how the livestream works.  Could someone point me in the right direction on where to read up?  Right now I'm confused as I dont know if I should be entering the UDP server in the args/gui tool, if I need to be creating an RTMP server or doing something special with VLC.

FWIW, I have been able to successfully connect to the gopro using my test scripts and can control shutter, settings, transfer files all just fine.  Live streaming is my sticking point.

The farthest I think I have gotten is using the gui tool where I have sucessfully registered to live stream, and have been able to make a request to start the stream, but then I get an error response (the livestream request appears successful, but there's an accompanying error response for the shutter).

Many Thanks.

**Comment by tcamise-gpsw** [comment]
Hello. If you want to test out a turn-key demo, you can use the Dockerized RTMP server from the script here: https://github.com/gopro/OpenGoPro/blob/main/demos/python/sdk_wireless_camera_control/tools/start_rtmp_server.sh
